Misc/NEWS.d/next/Documentation/2018-06-21-01-29-36.bpo-33894.tsesJ.rst

```diff
@@ -0,0 +1 @@
+Clarified the `tempdir` documentation.
```

Comment:
I don't think this merits a NEWS entry. Let's delete this file.

Reply:
Done.